common/src/main/java/io/pravega/common/tracing/RequestTracker.java

```diff
@@ -38,8 +38,10 @@
 @Slf4j
 public final class RequestTracker {
 
+    @VisibleForTesting
+    static final int MAX_PARALLEL_REQUESTS = 10;
     private static final String INTER_FIELD_DELIMITER = "-";
-    private static final int MAX_CACHE_SIZE = 1000000;
+    private static final int MAX_CACHE_SIZE = 100000;
```

Comment:
Does it make sense to make it a configurable property ?

Reply:
This has not been a problem in severa years this has been set to that value, and even though I just reduced it by 10x lower based on the reviews. We could create a parameter for that in a separate PR, but I doubt that this is going to be used by anyone nor solve the problem we are discussing in this PR.